packages/trader/src/Modules/SmartChart/Components/buy-toast-notification.jsx

```diff
@@ -0,0 +1,57 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { MobileWrapper, Toast, Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+import { findContractCategory } from '../../Trading/Helpers/contract-type';
+
+const BuyToastNotification = ({ portal_id = 'popup_root', action_toastbox, actionChangeToastbox }) => {
+    const { buy_price, currency, contract_type, list } = action_toastbox;
+    const active_trade_type = { value: contract_type };
+
+    const trade_type_name = findContractCategory(list, active_trade_type)?.contract_types?.find(
+        item => item.value === contract_type
+    ).text;
+
+    const message = (
+        <Text as='p' size='xxs' className='dc-toast__notification'>
```

Comment:
You can conditionally change text size for mobile and desktop.

Reply:
Its only for Mobile View